miqa/core/models/project.py

```diff
@@ -69,14 +69,14 @@ def clean(self):
 
         super().clean()
 
-    def get_read_permission_groups():
+    def get_read_permission_groups(self):
         return ['collaborator', 'tier_1_reviewer', 'tier_2_reviewer']
 
-    def get_review_permission_groups():
+    def get_review_permission_groups(self):
         return ['tier_1_reviewer', 'tier_2_reviewer']
 
     def get_user_role(self, user):
-        perm_order = Project.get_read_permission_groups()
+        perm_order = Project().get_read_permission_groups()
```

Comment:
```suggestion
        perm_order = self.get_read_permission_groups()
```

We should either use `self` or simply make the other methods `@staticmethods`

Reply:
Changed in [4f69033](https://github.com/OpenImaging/miqa/pull/437/commits/4f690338b966368939da389ca629d221346ca243)